exec/main.c

```diff
@@ -1312,6 +1404,15 @@ int main (int argc, char **argv, char **envp)
 		corosync_exit_error (COROSYNC_DONE_EXIT);
 	}
 
+
+	/*
+	 * Try to move corosync into root cpu cgroup. Failure is not fatal and
+	 * error is deliberately ignored.
+	 */
+	if (move_to_root_cgroup) {
+		(void)corosync_move_to_root_cgroup();
+	}
```

Comment:
No real advantage, but SCHED_RR and root cgroup are not equal, so even it's pretty nonsense it allows SCHED_OTHER + root cgroup.